guides/common/assembly_updating-server-smart-proxy-hosts.adoc

```diff
@@ -0,0 +1,17 @@
+ifdef::context[:parent-context: {context}]
+
+[id="Updating_Server_SmartProxy_Content_Hosts_{context}"]
+= Updating {ProjectServer} and {SmartProxyServerTitle}
+
+Use this chapter to update your existing {ProjectServer} and {SmartProxyServer} to a new patch version, for example, from {ProjectVersion}.0 to {ProjectVersion}.1.
+
+Updates patch security vulnerabilities and minor issues discovered after code is released, and are often fast and non-disruptive to your operating environment.
+
+Before updating, back up your {ProjectServer} and all {SmartProxyServers}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
```

Comment:
Perhaps we could include the backup and restore assemblies in this guide instead of refering to another guide?

Reply:
IMO, we should re-direct end users. Keeping backup guide in front of them may give a sense of doubt and hesitation to update.